front/app/components/admin/DatePickers/DateRangePicker/index.tsx

```diff
@@ -52,10 +52,14 @@ const DateRangePicker = ({
           selectedRange={selectedRange}
           selectionMode={selectionMode}
           onClickFrom={() => {
-            setSelectionMode('from');
+            selectionMode === 'from'
```

Comment:
Make it possible to close the dropdown without clicking outside (easier for e2e tests)

Reply:
Yeah something got screwed up here, it's not supposed to work like this. Not sure what happened but will fix on a separate branch

Reply:
Actually I'm not sure if I can reproduce this issue. Where are you using the date picker?